CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ set_property( CACHE CMAKE_CUDA_ARCHITECTURES PROPERTY STRINGS
 #-----------------------------------
 # BLAS options
 # SLATE defaults to single-threaded BLAS.
-set( blas_threaded "false" CACHE STRING
+set( blas_threaded "auto" CACHE STRING
```

Comment:
Update comment above to match code, or just remove comment altogether.

Reply:
Does the GNUmakefile need a similar change? Currently it has `blas_threaded ?= 0` at line 60. (Which GitHub doesn't allow me to tag because it isn't near where you edited.)

Reply:
Also update INSTALL.md to change `blas_threaded` default.

Note that I don't think we can use multi-threaded ESSL, or at least it isn't tested. While MKL recognizes when it is inside an OpenMP parallel region, I don't know that ESSL does that. So maybe we need a smarter blas_threaded option that is `auto` for SYCL and `false` otherwise.

Reply:
Also update INSTALL.md to add `sycl` as `gpu_backend` option.